support/docker/production/.env

```diff
@@ -3,7 +3,10 @@ PEERTUBE_DB_PASSWORD=postgres_password
 PEERTUBE_WEBSERVER_HOSTNAME=domain.tld
 PEERTUBE_WEBSERVER_PORT=443
 PEERTUBE_WEBSERVER_HTTPS=true
+# If you need more than one IP as trust_proxy
+# pass them as an space separated list:
```

Comment:
"as _a_ space separated list"

Reply:
Woops, sorry for my poor English :'(
I'll fix that asap